db/migrate/20230512070357_update_questionnaires.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateQuestionnaires < ActiveRecord::Migration[6.1]
+  def up
+    add_column :questionnaires, :description, :string
+    change_column :questionnaires, :dsl_version, :decimal, precision: 4, scale: 2
```

Comment:
We can change column from integer to decimal without losing data or affecting existing code. Won't lead to any crashes, we've tested locally.